### ISSUE TITLE
this file Annotations.md  has a small mistake !

### ISSUE THREAD

**Issue by Xiahl1990** [AngularClass/awesome-angular2#119]
eg:
```
@Directive({
  selector: '[tooltip]',     | CSS Selector which triggers the decorator
  inputs: [                  | List which properties need to be bound
    'text: tooltip'          |  - DOM element tooltip property should be
  ],                         |    mapped to the directive text property.
  host: {                    | List which events need to be mapped.
    (mouseover): 'show()'    |  - Invoke the show() method every time
  }                          |    the mouseover event is fired.
})                           |
class Form {                 | Directive controller class, instantiated
                             | when CSS matches.
  text:string;               | text property on the Directive Controller.
                             |
  show(event) {              | Show method which implements the show action.
    console.log(this.text);  |
  }
}
```

 `(mouseover): 'show()'`  should be `'(mouseover)': 'show()'`

**Comment by mgechev** [comment]
@Xiahl1990 would you open a PR with the suggested fix?

**Comment by rolandjitsu** [comment]
Closed by 2295e40a9cf804bf3691f3208199581061d9c826
